tests/test_module.py

```diff
@@ -473,6 +473,10 @@ def test_range_with_agg_query_max(self):
                                               'SUM', 4)
             assert expected_result == actual_result
 
+            actual_result = r.execute_command('TS.range', 'tester', 5, 8, 'AGGREGATION',
```

Comment:
@ashtul added the test to verify and assess that it does work with a number of samples lower than the bucket size.